docker-compose.yml

```diff
@@ -77,6 +79,8 @@ services:
     image: redis:7.2.2-alpine3.18
     networks:
       - opi
+    ports:
```

Comment:
why expose the port here ?

Reply:
If I want to run a server without a container, I need to provide an environment as well. I can type `docker-compose up redis` and get redis listening on the port. The same with `jaeger`. But if you want, I can remove this part.